sig-robotics/examples/Generating-Action-Sequences-for-Robot-Task-Plans-using-LLM/src/bot_description/setup.py

```diff
@@ -0,0 +1,32 @@
+from setuptools import setup
+from glob import glob
+import os
+
+package_name = "bot_description"
+
+setup(
+    name=package_name,
+    version="0.0.0",
+    packages=[package_name],
+    data_files=[
+        ("share/ament_index/resource_index/packages", ["resource/" + package_name]),
+        ("share/" + package_name, ["package.xml"]),
+        (os.path.join("share", package_name, "launch"), glob("launch/*.launch.py")),
+        (os.path.join("share", package_name, "urdf"), glob("urdf/**")),
+        (os.path.join("share", package_name, "world"), glob("world/**")),
+    ],
+    install_requires=["setuptools", "promptulate"],
+    zip_safe=True,
+    maintainer="zeeland",
+    maintainer_email="zeeland@foxmail.com",
```

Comment:
Respecting others' contributions is a fundamental principle of open source. Please avoid invasive changes. You can run your node in a separate terminal while keeping the FishROS node running in another terminal. This way, both nodes can operate independently without interference.